exercises/saddle-points/example.rs

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
this empty line *could* be removed, and I think it should be. The fact that there aren't any `use` lines shouldn't affect things I don't think, because if they were present, we wouldn't start the file with an empty line either.